Index: src/main/java/aar226_akc55_ayc62_ahl88/asm/Expressions/ASMMemExpr.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package aar226_akc55_ayc62_ahl88.asm.Expressions;\n\npublic class ASMMemExpr {\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/aar226_akc55_ayc62_ahl88/asm/Expressions/ASMMemExpr.java b/src/main/java/aar226_akc55_ayc62_ahl88/asm/Expressions/ASMMemExpr.java
--- a/src/main/java/aar226_akc55_ayc62_ahl88/asm/Expressions/ASMMemExpr.java	
+++ b/src/main/java/aar226_akc55_ayc62_ahl88/asm/Expressions/ASMMemExpr.java	
@@ -1,4 +1,14 @@
 package aar226_akc55_ayc62_ahl88.asm.Expressions;
 
-public class ASMMemExpr {
+import aar226_akc55_ayc62_ahl88.src.polyglot.util.InternalCompilerError;
+
+public class ASMMemExpr extends ASMExpr{
+    ASMExpr address;
+
+    public ASMMemExpr(ASMExpr inside){
+        if (inside instanceof ASMMemExpr){
+            throw new InternalCompilerError("memory operand inside of memory");
+        }
+        address = inside;
+    }
 }
